### PR TITLE
Fix assign style

### LINKED ISSUE #41: `assignStyleToLayer` needs default value for `isDefaultStyle`
see [code](https://github.com/meggsimum/geoserver-node-client/blob/master/src/style.js#L167).

Either documentation or code is wrong. It is required to provide `isDefaultStyle`

### workaround

Currently it is required to provide **all** arguments even though the documentation says something different

### PR DESCRIPTION
**Pull request by JakobMiksch**
- fixes #41
- default behavior is set to use the assigned style as default the default style of the layer
- swap argument order: 1) workspace 2) style - to be consistent with other functions

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -432,8 +432,31 @@ describe('style', () => {
...
+
+    await grc.styles.assignStyleToLayer(workSpace, featureLayerName, workspaceStyle, styleName, isDefaultStyle);
+
+    const layerInfo2 = await grc.layers.get(workSpace, featureLayerName)
+    const defaultStyleNameAfter = layerInfo2.layer.defaultStyle.name;
+    // we check if the default style of before has not changed
+    expect(defaultStyleNameBefore).to.equal(defaultStyleNameAfter);
+
+    const associatedStyleName = layerInfo2.layer.styles.style[0].name;
+    const InputStyleQualifiedName = `${workspaceStyle}:${styleName}`
```
Comment (chrismayer) [inline test/test.js:449]: Why uppercase notation?

**Review by chrismayer — COMMENTED** [review]
One minor question, which should be clarified first.

File: test/test.js (hunk 1 above)
Comment (JakobMiksch) [reply test/test.js]: good catch, just a typo
fixed with [31b189b](https://github.com/meggsimum/geoserver-node-client/pull/67/commits/31b189b0ac5029ce5ba68a63402a84d5f251f6a7)

**Comment by JakobMiksch** [comment]
@chrismayer I fixed the typo in the code and pulled the latest changes from the master. The PR is ready for review.

**Review by chrismayer — APPROVED** [review]
LGTM now. Please merge at will.

**Review by chrismayer — APPROVED** [review]
Thanks for this, LGTM.

**Pull request merged**